Dockerfile

```diff
@@ -29,24 +42,16 @@ RUN echo '5a70f1f3db0608c156d5b6629d4cbc3b304fc045 /etc/systemd/system/sssd.serv
 RUN find /etc/systemd/system/* '!' -name '*.wants' | xargs rm -rvf
 RUN for i in basic.target sysinit.target network.service netconsole.service ; do rm -f /usr/lib/systemd/system/$i && ln -s /dev/null /usr/lib/systemd/system/$i ; done
 
-RUN echo LANG=C > /etc/locale.conf
-
-RUN /sbin/ldconfig -X
-
 COPY init-data ipa-server-configure-first ipa-server-status-check exit-with-status ipa-volume-upgrade-* /usr/sbin/
-RUN chmod -v +x /usr/sbin/init-data /usr/sbin/ipa-server-configure-first /usr/sbin/ipa-server-status-check /usr/sbin/exit-with-status /usr/sbin/ipa-volume-upgrade-*
 COPY container-ipa.target ipa-server-configure-first.service ipa-server-upgrade.service ipa-server-update-self-ip-address.service /usr/lib/systemd/system/
 RUN rmdir -v /etc/systemd/system/multi-user.target.wants \
-	&& mkdir /etc/systemd/system/container-ipa.target.wants \
-	&& ln -s /etc/systemd/system/container-ipa.target.wants /etc/systemd/system/multi-user.target.wants
+    && mkdir /etc/systemd/system/container-ipa.target.wants \
+    && ln -s /etc/systemd/system/container-ipa.target.wants /etc/systemd/system/multi-user.target.wants
 
-RUN systemctl set-default container-ipa.target
-RUN systemctl enable ipa-server-configure-first.service
+RUN systemctl set-default container-ipa.target \
+    && systemctl enable ipa-server-configure-first.service
```

Comment:
Again, putting together two commands ... and I've just file https://bugzilla.redhat.com/show_bug.cgi?id=1340825 because of bug in rawhide, that `set-default` now does not return 0. Having them in one `RUN` makes errors less obvious.